templates/desktop/index.html

```diff
@@ -106,7 +111,7 @@ <h2>
     </div>
     <div class="col-5 col-start-large-8 u-vertically-center u-hide--medium u-hide--small">
       {{ image (
-        url="https://assets.ubuntu.com/v1/fa57dc30-desktop-22.04-laptop.png",
+        url="https://assets.ubuntu.com/v1/a8e96807-desktop-xps13-small.png",
         alt="Laptop running Ubuntu",
         width="1000",
         height="578",
```

Comment:
Looks good, but we should update these dimensions to match the new image (not a huge difference but we've had issues with distorted images):
```suggestion
        width="1088",
        height="630",
```